modules/sdk/gradle-plugins-workspace/src/test/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtensionValidatorTest.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.gradle.plugins.workspace.internal.client.extension;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.Files;
+
+import java.util.HashMap;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.Project;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Anderson Luiz
+ */
+public class ClientExtensionValidatorTest {
+
+	@Test
+	public void testShouldValidateSuccessfullyIfNoFrontendTokenDefinitionProvided() {
+		ClientExtension clientExtension = new ClientExtension();
+
+		clientExtension.type = "themeCSS";
+		clientExtension.typeSettings = new HashMap<String, Object>() {
+		};
+
+		Project project = Mockito.mock(Project.class);
+
+		ClientExtensionValidator clientExtensionValidator =
```

Comment:
You're creating a `ClientExtensionValidator` in every test, please create a final attribute for that.